tests/syscall/fs/host/host.c

```diff
@@ -14,48 +17,55 @@
 
 #define SKIP_RETURN_CODE 2
 
-int rmdir(const char* path);
-
-int test_fs_posix(
-    const char* enclave_path,
-    const char* src_dir,
-    const char* tmp_dir)
+#if defined(_WIN32)
```

Comment:
Since this file now has approximately 0 lines of code in common between Windows and Linux, I might suggest splitting it into two files, like host_win32.c and host_linux.c for example.  One benefit is that it then becomes somewhat easier to do a diff between the two files to see what the platform-specific differences are, and to help verify that they both do the same thing.

Reply:
Yes two files and no macros.